src/containers/PageHeader/index.js

```diff
@@ -28,20 +28,29 @@ PageHeaderContainerComponent.defaultProps = {
 };
 
 PageHeaderContainerComponent.propTypes = {
-  activeFilter: PropTypes.shape({
+  filter: PropTypes.shape({
     name: PropTypes.string,
+    options: PropTypes.object,
```

Comment:
You probably want to specify these (maybe put the proptype definition is separate file)

Reply:
gedaan